setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='flaskutils',
-    version='2.1.0',
+    version='2.2.0',
```

Comment:
I forgot to increment the version here when I created the last release, could you update this to 2.3.0?